File: C programming/Practice C/palindrome.c
#include <stdio.h>

int main()
{
    int num, reverse = 0, r, t;
    printf("Enter a number\n");
    scanf("%d", &num);
    t = num;
    while (t != 0)
    {
        r = t % 10;
        reverse = reverse * 10 + r;
        t = t / 10;
    }
    if (num == reverse)
        printf("%d is palindrome", num);
    else
        printf("%d is not palindrome", num);
    return 0;
}

File: JavaRevise/src/PersonalTesting/Palindrome.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package PersonalTesting;

import java.util.Scanner;

/**
 *
 * @author Himanish
 */
public class Palindrome {

    public static void main(String[] args) {
        int temp, sum = 0, rem;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number to check Palindrome");
        int num = sc.nextInt();
        temp = num;
        while (temp > 0) {
            rem = temp % 10;
            sum = (sum * 10) + rem;
            temp = temp / 10;
        }
        if (sum == num) {
            System.out.println(num + " is a palindrome number");
        } else {
            System.out.println(num + " is not a palindrome number");
        }
    }
}
